Makefile

```diff
@@ -51,18 +51,26 @@ _output/share/lima/lima-guestagent.Linux-aarch64:
 	chmod 644 $@
 
 .PHONY: install
+install: | $(abspath $(DESTDIR)/$(PREFIX))
+install: DEST = $(realpath $(DESTDIR)/$(PREFIX))
 install:
-	cp -av _output/* "$(DESTDIR)/$(PREFIX)/"
-	if [[ $(shell uname -s ) != Linux && ! -e "$(DESTDIR)/$(PREFIX)/bin/nerdctl" ]]; then ln -sf nerdctl.lima "$(DESTDIR)/$(PREFIX)/bin/nerdctl"; fi
+	@test -d "$(DEST)" || (echo "invalid destination: $(DESTDIR)/$(PREFIX)"; exit 1)
```

Comment:
Given that the directory has just been created, this should never actually trigger. If `mkdir -p` cannot create the directory, it will already fail:

```console
$ mkdir -p /bin/foo;echo $?
mkdir: /bin/foo: Operation not permitted
1
```

And if the directory does already exist, but is not writable, then the `cp` command is going to fail with a relevant error message.

So I think the test serves no useful purpose and should be removed.